esmvalcore/preprocessor/_io.py

```diff
@@ -191,6 +203,7 @@ def _load_zarr(
             consolidated=True,
             decode_times=time_coder,
             engine="zarr",
+            chunks={},
```

Comment:
One thing you could do to avoid duplicating these keyword arguments is to define

```py
open_kwargs = {
    "consolidated": False,
    "engine": "zarr",
    "chunks": {},
    "backend_kwargs": backend_kwargs,
}
```

at the top and then update those for the remote store. Then you can simply do a

```py
zarr_xr = xr.open_dataset(file, **open_kwargs)
```

after the if/else

Reply:
a very good idea, thanks, Manu! Done in d6289481447bbf77dc51834f2f40a51e32d004c6